### DISCUSSION TITLE
Different path app/Filament

### DISCUSSION THREAD

**Discussion by uadoronin** [filamentphp/filament#3635] [Help]
Hi!

Filament has a configuration file where you can configure namespaces and paths.
I changed the default folder from `app/Filament` to `app/Admin`, and specified all the namespaces and paths in the configuration file.
But if you run the `artisan filament:resource ...` command or other, the resource is automatically created at the old `app/Filament` path.

Maybe you need to change the default Make Commands, and take the paths from the configuration file when generating it. Or perhaps I'm wrong?

**Comment by danharrin** [comment]
Unfortunately it sounds much easier than it actually is to implement, we've tried to find a nice solution several times. But if you use an IDE like PHPStorm then it should be quite easy to move the classes to the correct place
